client/src/Carousel.jsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
 import T from 'prop-types';
 import ENV from './config';
+import Item from './Item';
 
-const Carousel = ({ items, position, onItemClick }) => (
-  <div data-position={position}>
-    {items.map((item) => <button type="button" key={item.id} data-id={item.id} onClick={onItemClick}>{item.name}</button>)}
-  </div>
+const Carousel = ({
+  items, position, onItemClick, title,
+}) => (
+  <>
+    <div className="g-carousel-title">{title}</div>
+    <div data-position={position} className="g-carousel-container">
+      <button type="button" className="g-btn g-arrow-holder" disabled={position === 0} data-dir="-1">
+        <div className="g-arrow left" />
```

Comment:
I noticed you used the class names left and right for a few things--this is a potential issue as a raw name if anyone else has done the same.